core/services/chainlink/testdata/config-full.toml

```
@@ -0,0 +1,290 @@
+UseBatchSend = true
+
+[Log]
+JSONConsole = true
+FileDir = 'log/file/dir'
+SQL = true
+FileMaxSize = '100.00gb'
+FileMaxAgeDays = 17
+FileMaxBackups = 9
+UnixTS = true
+
+[WebServer]
+AllowOrigins = '*'
+ExternalURL = 'https://bridge.response'
+HTTPWriteTimeout = '1m0s'
+HTTPPort = 56
+SecureCookies = true
+SessionTimeout = '1h0m0s'
+
+[WebServer.MFA]
+RPID = 'test-rpid'
+RPOrigin = 'test-rp-origin'
+
+[WebServer.RateLimit]
+Authenticated = 42
+AuthenticatedPeriod = '1s'
+Unauthenticated = 7
+UnauthenticatedPeriod = '1m0s'
+
+[WebServer.TLS]
+CertPath = 'tls/cert/path'
+ForceRedirect = true
+Host = 'tls-host'
+HTTPSPort = 6789
+KeyPath = 'tls/key/path'
+
+[JobPipeline]
+DefaultHTTPRequestTimeout = '1m0s'
+ExternalInitiatorsEnabled = true
+HTTPRequestMaxSizeBytes = 67
+MaxRunDuration = '1h0m0s'
+ReaperInterval = '4h0m0s'
+ReaperThreshold = '168h0m0s'
+ResultWriteQueueDepth = 10
+
+[FluxMonitor]
+DefaultTransactionQueueDepth = 100
+SimulateTransactions = true
+
+[OCR2]
+ContractConfirmations = 11
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h0m0s'
+ContractSubscribeInterval = '1m0s'
+ContractTransmitterTransmitTimeout = '1m0s'
+DatabaseTimeout = '8s'
+KeyBundleID = '7a5f66bbe6594259325bf2b4f5b1a9c900000000000000000000000000000000'
+MonitoringEndpoint = 'test-mon-end'
+
+[OCR]
+ObservationTimeout = '11s'
+BlockchainTimeout = '3s'
+ContractPollInterval = '1h0m0s'
+ContractSubscribeInterval = '1m0s'
+DefaultTransactionQueueDepth = 12
+KeyBundleID = 'acdd42797a8b921b2910497badc5000600000000000000000000000000000000'
+MonitoringEndpoint = 'test-monitor'
+SimulateTransactions = true
+TransmitterAddress = '0xa0788FC17B1dEe36f057c42B6F373A34B014687e'
+
+[P2P]
+IncomingMessageBufferSize = 13
+OutgoingMessageBufferSize = 17
+
+[P2P.V1]
+AnnounceIP = '1.2.3.4'
+AnnouncePort = 1234
+BootstrapCheckInterval = '1m0s'
+DefaultBootstrapPeers = ['foo', 'bar', 'should', 'these', 'be', 'typed']
+DHTAnnouncementCounterUserPrefix = 4321
+DHTLookupInterval = 9
+ListenIP = '4.3.2.1'
+ListenPort = 9
+NewStreamTimeout = '1s'
+PeerID = '12D3KooWMoejJznyDuEk5aX6GvbjaG12UzeornPCBNzMRqdwrFJw'
+PeerstoreWriteInterval = '1m0s'
+
+[P2P.V2]
+AnnounceAddresses = ['a', 'b', 'c']
+DefaultBootstrappers = ['1', '2', '3']
+DeltaDial = '1m0s'
+DeltaReconcile = '1s'
+ListenAddresses = ['foo', 'bar']
+
+[Keeper]
+CheckUpkeepGasPriceFeatureEnabled = true
+DefaultTransactionQueueDepth = 17
+GasPriceBufferPercent = 12
+GasTipCapBufferPercent = 43
+BaseFeeBufferPercent = 89
+MaximumGracePeriod = 31
+RegistryCheckGasOverhead = '90'
+RegistryPerformGasOverhead = '18446744073709551615'
+RegistrySyncInterval = '1h0m0s'
+RegistrySyncUpkeepQueueSize = 31
+TurnLookBack = 91
+TurnFlagEnabled = true
+
+[AutoPprof]
+Enabled = true
+ProfileRoot = 'prof/root'
+PollInterval = '1m0s'
+GatherDuration = '12s'
+GatherTraceDuration = '13s'
+MaxProfileSize = '1.00gb'
+CPUProfileRate = 7
+MemProfileRate = 9
+BlockProfileRate = 5
+MutexProfileFraction = 2
+MemThreshold = '1.00gb'
+GoroutineThreshold = 999
+
```

@samsondav Should we add a sentry section?
```suggestion

[Sentry]
Debug = true
DSN = ''
Environment = ''
Release = ''

```